### PR TITLE
Phase out current Upstream docs

### PR DESCRIPTION
**Pull request by joelhans**
This PR hides the existing Upstream-related documentation—everything under the **Using Radicle** and **Connecting to Ethereum** headings—by removing them from the sidebar navigation. They're still in this repo so that we can re-use them later if need be.

I'm also introducing a new version of the getting started guide that focuses on installing the CLI tools and the first few steps to hosting a repo on the Radicle network, like creating an identity and initializing a project.

Read more about why I'm making this change [here](https://www.notion.so/commitcopy/docs-radicle-xyz-planning-architecture-27de5273d4d34193b4de114d721cabc7#98ecd3f0d7924c3da67a6493086b793e).

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -47,135 +51,164 @@ Radicle development is public by default and completely open-source. See
   </em>
 </details>
 
-### Supported Platforms
+## Install the Radicle CLI
 
-Radicle Upstream is available for Linux and macOS (version 10.14 and
-newer).
+The Radicle CLI installs on Linux, macOS, Windows-based systems by compiling from source or with Homebrew.
```
Comment (rudolfs) [inline docs/getting-started.md:56]: Does it really work on Windows? Has anyone tested that?

File: docs/getting-started.md [hunk 2]
```diff
@@ -47,135 +51,164 @@ Radicle development is public by default and completely open-source. See
...
+
+After you've pushed your project to the Radicle network, it's available for viewing on the Radicle [web
+client](https://app.radicle.network).
+
+To view your project, paste the URL after `(web)` into your browser of choice. You can use this interface to browse your
+code, view individual files, explore your commit history, and more. You can share this URL with others, which allows
+them to view your code on the web client, clone your code into a new project.
+
+![The Radicle web client for viewing a published
+project](https://user-images.githubusercontent.com/1153921/157982795-0bfcaa58-ec9d-44f2-8da3-ad0900172e0e.png)
```
Comment (rudolfs) [inline docs/getting-started.md:194]: We should add this image to this repo and just use a relative link here.

**Review by rudolfs — COMMENTED** [review]
Other than the 2 inline comments this looks good to me. I assume the "Troubleshooting", "FAQ" and "Glossary" sections will be reworked in a separate step?

File: docs/getting-started.md (hunk 1 above)
Comment (joelhans) [reply docs/getting-started.md:56]: I just asked on Discord—we'll see if I get a response.

File: docs/getting-started.md (hunk 2 above)
Comment (joelhans) [reply docs/getting-started.md]: Fixed! Old habits die hard.

**Comment by joelhans** [comment]
@rudolfs Exactly—the idea is to merge this PR when it's ready and then go through those few documents for a follow-up, which I can hopefully get into a second PR tomorrow.

**Review by rudolfs — APPROVED** [review]

**Pull request merged**